pkg/leanix/integration_hub.go

```diff
@@ -46,8 +47,10 @@ var log = logging.MustGetLogger("leanix-k8s-connector")
 
 // SelfStartRun initiates the Integration Hub run and response with id
 func SelfStartRun(fqdn string, accessToken string, datasource string) (*SelfStartResponse, error) {
-	datasourceRunUrl := "https://" + fqdn + "/services/integration-hub/v1/datasourceRuns/name/" + datasource + "/selfStart"
-	req, err := http.NewRequest("GET", datasourceRunUrl, nil)
+	datasourceRunUrl := fmt.Sprintf("https://%s/services/integration-hub/v1/datasources/%s/selfStart", fqdn, datasource)
+	req, err := http.NewRequest("POST", datasourceRunUrl, nil)
+	req.Header.Set("connector-version", version.VERSION)
```

Comment:
Did you try this request out manually with an access token and a test datasource so that we can be sure, that it is correct?

Reply:
Yeah so the good part here is when I ran the code, iHub pushes the data to Log analytics. The Azure workbook mentioned in PR has the details of my workspace and version. thats also part of the Acceptance Criteria; Can you also please look into that